compiler/rustc_feature/src/active.rs

```diff
@@ -504,9 +507,6 @@ declare_features! (
     (active, never_type_fallback, "1.41.0", Some(65992), None),
     /// Allows `#![no_core]`.
     (active, no_core, "1.3.0", Some(29639), None),
-    /// Allows function attribute `#[no_coverage]`, to bypass coverage
-    /// instrumentation of that function.
-    (active, no_coverage, "1.53.0", Some(84605), None),
```

Comment:
should probably add `no_coverage` to the renamed features list?

Reply:
Didn't notice that was a thing, will do.

Reply:
Something like https://github.com/rust-lang/rust/blob/9a5ac560947a4e7cf0884868c3d83ab18c68ec45/compiler/rustc_feature/src/removed.rs#L98-L99 should do it

Reply:
Hmmm, I've written something, but I have two questions:

1) What release would it make sense to put?  I'm guessing it's logically 1.72.0?
2) There's a proposal above to accept both features/attributes (emitting a lint for the old one) which presumably would mean _not_ adding a "removed feature" entry until the deprecated feature is actually removed?  OTOH this is an unstable feature so maybe the policy is to just remove them?  We know there are crates in the real world (https://docs.rs/coverage-helper/latest/coverage_helper/ for example) that use the unstable feature today which would be broken by this change as it stands.

Reply:
> * What release would it make sense to put?  I'm guessing it's logically 1.72.0?

`CURRENT_RUSTC_VERSION`, it will be replaced on version bump.